tests/cypress/integration/newfold-labs/wp-module-onboarding/Design/header-menu.cy.js

```diff
@@ -0,0 +1,71 @@
+describe('Header menu Page', function () {
+
+    before(() => {
+        cy.exec('npx wp-env run cli wp theme activate yith-wonder');
+        cy.exec('npx wp-env run cli wp option update mm_brand BlueHost');
```

Comment:
We can go ahead without mm_brand too as that is gonna reduce time in running this Cypress Test.

Reply:
sure